practice/src/css/components/text.css

```diff
@@ -0,0 +1,27 @@
+.main-heading {
+  color: var(--primary-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-md);
+  line-height: var(--line-height-xl);
+}
+
+.secondary-heading {
+  color: var(--primary-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-lg);
+  line-height: var(--line-height-2xl);
+}
+
+.main-title {
```

Comment:
difference between title and heading?

Reply:
heading: talk about the main content of the page
title: talk about the main content of the card